src/utils/math.ts

```diff
@@ -7,3 +7,8 @@ export const getPercentile = (arr: number[], percentile: number): number => {
   const index = Math.floor((percentile / 100) * (arr.length - 1));
   return arr[index];
 };
+
+export const clamp = (
```

Comment:
From the name, I wouldn't expect this function to double the value.
`clamp` is pretty commonly used in math libraries to denote a function that clamps a value between min-max bounds. (eg: [css](https://www.codecademy.com/resources/docs/css/math-functions/clamp))

I'd suggest `double` or `doubleWithClamp`